src/sentry/models/organizationmapping.py

```diff
@@ -19,13 +19,12 @@ class OrganizationMapping(Model):
 
     __include_in_export__ = True
 
-    organization_id = BoundedBigIntegerField(db_index=True)
+    organization_id = BoundedBigIntegerField(db_index=True, unique=True)
```

Comment:
Yep! There should be no rows in this table at all but I'll double check that we've properly removed all write paths for this model. The addition of this uniqueness constraint is why I marked this as potentially dangerous in the event that I missed a path and we have dupes causing the migration to fail.

I'll go ahead and move this into its own PR anyway just in case.